endpoints/openrtb2/auction_test.go

```diff
@@ -3565,6 +3621,204 @@ var testStoredImps = []string{
 	``,
 }
 
+var testBidRequestForChannelTest = []string{
+	`{
```

Comment:
It looks like you only use elements with indexed 0, 1 and 3. If other data is not needed, can you please delete it? 
It's superhard to read this kind of inputs.

Reply:
Yes will do!